scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
As for the `local` at hand: if it is really supported everywhere (where relevant), this could be ignored with shellcheck then?!